src/test/java/uk/gov/hmcts/cmc/claimstore/documents/content/LegalSealedClaimContentProviderTest.java

```diff
@@ -35,4 +35,20 @@ public void shouldCreateContent() throws Exception {
         assertThat(contents).isNotEmpty().containsKey("feePaid").containsValue("£500.01");
     }
 
+    @Test
+    public void contentShouldIncludeWaterMarkFlag() throws Exception {
```

Comment:
this test doesn't look like it has a lot of value, you are hardcoding the value to be true, so of course its there

Reply:
also not sure you you aren't using the default fee amount / default SampleClaim

Reply:
used